### ISSUE TITLE
Moodle

### ISSUE THREAD

**Issue by rgaudin** [offspot/package-requests#14]
In GitLab by @Popolechien on Apr 3, 2019, 13:38

Moodle is a widely used educational instance, there's been some demand to have it made available.

**Comment by rgaudin** [comment]
I believe (from discussions with BSF) that every moodle integration will require a dedicated package, reusing a generic base. 
We should thus find an existing, useful moodle instance to build this base and first package.

**Comment by Popolechien** [comment]
pinging @martignoni who might have some valuable insights on this one.

**Comment by martignoni** [comment]
> I believe (from discussions with BSF) that every moodle integration will require a dedicated package, reusing a generic base. We should thus find an existing, useful moodle instance to build this base and first package.

If I understand correctly, it's just a matter of installing and configuring an empty Moodle server from the standard Moodle distribution.

**Comment by martignoni** [comment]
Indeed, and you've all of this: Moodle (GPL-3.0) and MoodleBox (AGPL-3.0) are free.

**Comment by Popolechien** [comment]
For reference: https://discuss.moodlebox.net/d/270-install-kiwix-on-moodlebox

**Comment by rgaudin** [comment]
@Popolechien I don't think this is useful for this ticket as it explains how to install Kiwix on a moodlebox…

**Comment by Popolechien** [comment]
Hi know, but I was sent this by Dan McGuire and figured it as good a way as any to put this back on top of the pile :-D

**Comment by rgaudin** [comment]
Exactly!
